src/core/Akka.TestKit/EventFilter/IEventFilterApplier.cs

```diff
@@ -123,6 +127,9 @@ public interface IEventFilterApplier
         /// <param name="action">The action.</param>
         /// <param name="cancellationToken"></param>
         Task ExpectAsync(int expectedCount, TimeSpan timeout, Func<Task> action, CancellationToken cancellationToken = default);
+        
+        [Obsolete("Use ExpectAsync<T>(expectedCount, TimeSpan, Func<Task<T>>) instead. This method only exists to support backwards compatibility as of Akka.NET v1.5.")]
```

Comment:
Added `Obsolete` APIs for `__Async` methods that were introduced in the v1.4 TestKit that accepted `Action` overloads.